dist/logic.js

```diff
@@ -0,0 +1,59 @@
+class WeatherManager {
+    constructor() {
+        this.cityData = []
+    }
+
+    fixer = function(myStr) {
```

Comment:
you are not supposed to declare things like this in a class, you should be using methods (do not do fixer = ...)

editString (cityName){
}